addon-test-support/setup-global-sinon.js

```diff
@@ -0,0 +1,14 @@
+import { createSandbox, restoreSandbox } from './sinon-sandbox';
+
+/**
+ * Allows for creating and restoring a global sinon sandbox per test. This is
+ * done via the `QUnit.testStart` and `QUnit.testDone` methods.
+ *
+ * @export
+ * @param {Object} An object containing optional options
+ * @public
+ */
+export default function setupSinon(testEnvironment = self.QUnit) {
```

Comment:
Should we rename this to `setupGlobalSinonRestoration` and add an export in `index.js`?

Reply:
I've actually been thinking about restructuring our imports to match what `ember-qunit` does, so you can do `import blah from 'ember-sinon-qunit`;` without needing `/test-support` at all. Wanted to get everything working as-is before I mucked with it, though.

Reply:
Sounds good.